screens/PrepareFace.tsx

```diff
@@ -1,85 +1,90 @@
-import { useNavigation } from '@react-navigation/native';
-import { Camera } from 'expo-camera'
-import React, { FunctionComponent, useEffect, useState } from 'react';
-import { StyleSheet, View } from 'react-native';
-import { Button, Text } from '../components/Themed';
-import { Ionicons }  from '@expo/vector-icons';
-
+import { useNavigation } from "@react-navigation/native";
+import { Camera } from "expo-camera";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { Button, Text } from "../components/Themed";
+import { Ionicons } from "@expo/vector-icons";
```

Comment:
if we're using prettier, we shoudl specify a config somewhere, but that can be another PR (add .prettierrc and format all files)

Reply:
Let's talk about this tomorrow at standup.